tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/remote/RemoteUtil.java

```diff
@@ -38,4 +43,39 @@ public static IRemoteServer getRemoteServer( IServer server )
 
         return null;
     }
+
+    public static boolean canConnect( String host, int port )
```

Comment:
There is already a method in the "SocketUtil" class.  Lets use that one instead of adding new one here.  If the method in "SocketUtil" doesn't work for your use-case you can modify it to make it work.